packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -34,6 +35,7 @@
 class Convert {
 
   private static BitmapDescriptor toBitmapDescriptor(Object o) {
+    final FlutterLoader flutterLoader = FlutterInjector.instance().flutterLoader();
```

Comment:
This probably isn't present on stable, or if it is, we have to increase the minimum version Flutter SDK constraint most likelyt o capture it. /cc @xster

Reply:
We should also consider whether we just ignore a depprecation warning for this or not - xster will have more context there.

Reply:
phew, glad you caught me. Forgot about targeting stable. We should indeed leave an ignore like https://github.com/flutter/plugins/pull/3072/files#diff-e57a1a7ffb729ad174b597626202429c1c87e4168e19b2aa9decc1be84e8777bR56 with a TODO to remove when the next stable ships. For now we have to use the old APIs.